packages/contracts/contracts/maci/MACI.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+import { MACI as BaseMACI } from "maci-contracts/contracts/MACI.sol";
+import { IPollFactory } from "maci-contracts/contracts/interfaces/IPollFactory.sol";
+import { IMessageProcessorFactory } from "maci-contracts/contracts/interfaces/IMPFactory.sol";
+import { ITallyFactory } from "maci-contracts/contracts/interfaces/ITallyFactory.sol";
+import { InitialVoiceCreditProxy } from "maci-contracts/contracts/initialVoiceCreditProxy/InitialVoiceCreditProxy.sol";
+import { SignUpGatekeeper } from "maci-contracts/contracts/gatekeepers/SignUpGatekeeper.sol";
+
+import { Poll } from "./Poll.sol";
```

Comment:
would it be more lightweight to import an interface to the poll with init and transferOwnership?